tests/composite_pk/test_get.py

```diff
@@ -152,50 +152,50 @@ def test_values_list(self):
         self.assertSequenceEqual(
             User.objects.values_list("pk").order_by("pk"),
             (
-                ((1, 1),),
-                ((1, 2),),
-                ((2, 3),),
+                (self.user_1.pk,),
+                (self.user_2.pk,),
+                (self.user_3.pk,),
             ),
         )
         self.assertSequenceEqual(
             User.objects.values_list("pk", "email").order_by("pk"),
```

Comment:
What happens if we do `values_list("pk", "id")`? I.e. the composite primary key and one of the component fields.